packages/squiggle-lang/src/rescript/Reducer/Reducer_Expression/Reducer_Expression.res

```diff
@@ -18,13 +18,12 @@ type internalCode = ReducerInterface_ExpressionValue.internalCode
 type t = expression
 
 /*
-  Converts a MathJs code to expression
+  Converts a Squigle code to expression
 */
-let parse_ = (expr: string, parser, converter): result<t, errorValue> =>
-  expr->parser->Result.flatMap(node => converter(node))
-
-let parse = (mathJsCode: string): result<t, errorValue> =>
-  mathJsCode->parse_(MathJs.Parse.parse, MathJs.ToExpression.fromNode)
+let parse = (peggyCode: string): result<t, errorValue> =>
+  peggyCode
+  ->Reducer_Peggy_Parse.parse
+  ->Result.map(node => Reducer_Peggy_ToExpression.fromNode(node))
```

Comment:
tiny thing: You could simplify the last line here:
``   ->Result.map(Reducer_Peggy_ToExpression.fromNode)``